react-ui/src/components/Header.js

```diff
@@ -6,15 +6,9 @@ export default class Header extends Component {
     render() {
 
       let header;
-      if(this.props.color==="red"){
```

Comment:
Removed these lines from header file as they are no longer used in our application.

Reply:
Appreciate the attention to detail to able to make the codebase current and concise!